supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc

```diff
@@ -174,7 +174,7 @@ Capitalize it only when it starts a sentence.
 
 [[identity-provider-federation]]
 ==== image:images/yes.png[yes] identity provider federation (noun)
-*Description*: In Red{nbsp}Hat Single Sign-On, you can delegate authentication to one or more IDPs, and this process is referred to as _identity provider federation_. Social login through Facebook or Google+ is an example of identity provider federation. You can also hook Red{nbsp}Hat Single Sign-On to delegate authentication to any other OpenID Connect or SAML 2.0 IDP.
+*Description*: In Red{nbsp}Hat Single Sign-On, you can delegate authentication to one or more IdPs, and this process is referred to as _identity provider federation_. Social login through Facebook or Google+ is an example of identity provider federation. You can also hook Red{nbsp}Hat Single Sign-On to delegate authentication to any other OpenID Connect or SAML 2.0 IdP.
```

Comment:
To remove the Vale flag rule for IdPs, update the following file to remove the "IDPs" entry:

.vale/styles/config/vocabularies/SSG/accept.txt#L70

This update can be done inside the [supplementary-style-guide](https://github.com/redhat-documentation/supplementary-style-guide) so you can include the update with this PR.